.svlint.toml

```diff
@@ -0,0 +1,37 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Can you add a comment along the lines of

Rules for svlint, a SystemVerilog linter commonly used in editors.
The configuration matches the lowRISC SystemVerilog style guide at
https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md.
See https://github.com/dalance/svlint/blob/master/RULES.md for a list of rules.

Reply:
SG.